database/SQL/helpers.js

```diff
@@ -1,3 +1,117 @@
 const db = require('./index.js');
 
-module.exports = {};
\ No newline at end of file
+var getProductList = async (count, page) => {
+	return new Promise (async (resolve, reject) => {
+		var result = [];
+		var start = ((page-1) * count) + 1;
+		for (var i = ((page-1) * count) + 1; i < start + count; i++) {
+			await db.query(`SELECT json_build_object(
+				'id', p.product_id,
+				'name', p.name,
+				'slogan', p.slogan,
+				'description', p.description,
+				'category', p.category,
+				'default_price', p.default_price
+				)
+				AS product
+				FROM product_list p
+				WHERE p.product_id = ${i}
+				GROUP BY p.product_id;`)
+				.then((data) => {
+					if (data.rows.length === 0) {
+						resolve(result);
+						return;
+					}
+					result.push(data.rows[0].product);
+				})
+				.catch((err) => {
+					reject(err);
+					return;
+				});
+		}
+		resolve(result);
+	});
+};
+
+var getProductInfo = async (id) => {
+	return new Promise ((resolve, reject) => {
+		db.query(`
+			SELECT json_build_object(
+				'id', p.product_id,
+				'name', p.name,
+				'slogan', p.slogan,
+				'description', p.description,
+				'category', p.category,
+				'default_price', p.default_price,
+				'features', (
+					SELECT COALESCE(json_agg(json_build_object(
+							'feature', f.feature,
+							'value', f.value
+					)), '[]')
+					FROM product_features f
+					WHERE p.product_id = f.product_id AND f.feature IS NOT NULL AND f.value IS NOT NULL
+				))
+			AS product
+			FROM product_list p
+			WHERE p.product_id = ${id};
+			`)
+			.then((data) => {
+				resolve(data.rows[0].product);
+			})
+			.catch((err) => {
+				reject(err);
+			});
+	});
+};
+
+var getProductStyles = (id) => {
+	return new Promise((resolve, reject) => {
+		db.query(`
```

Comment:
Apparently using a formatted string can open you up to SQL injections. You could look into using queries with variables such as,
`WHERE s.product_id = $1 GROUP BY s.product_id;', [id])`